aten/src/ATen/native/Distributions.cpp

```diff
@@ -172,6 +172,18 @@ Tensor& bernoulli_scalar_cpu_(Tensor& self, double p, Generator* gen) {
 }
 
 
+Tensor& exponential_cpu_(Tensor& self, double lambda, Generator* gen) {
+  AT_DISPATCH_ALL_TYPES(self.scalar_type(), "exponential_cpu_", [&] {
+    THGenerator* generator = get_generator(gen);
+    std::lock_guard<std::mutex> lock(generator->mutex);
+    CPU_tensor_apply1<scalar_t>(
```

Comment:
Did we merge already the PR that adds support for bernoulli with TensorIterator?

Reply:
It was done for CUDA but it is applicable for CPU https://github.com/pytorch/pytorch/pull/21300/files